docs/docs/APIGuide/Layers/Containers.md

```diff
@@ -432,7 +432,9 @@ import com.intel.analytics.bigdl.nn._
 import com.intel.analytics.bigdl.tensor.Tensor
 import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric.NumericFloat
 
-val model = Bottle(Linear(3, 2), 2, 2)
+val module = Sequential()
```

Comment:
Do not add a `sequential` here to avoid confusion.

you can just make the change to ->
```scala
val model = Bottle(Linear(3, 2).asInstanceOf[Module[Float]], 2, 2)
```